students/hangu/users/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+
+class User(models.Model):
+    user_name = models.CharField(max_length=10)
+    user_email = models.CharField(max_length=45)
+    password = models.CharField(max_length=20)
+    phone_number = models.CharField(max_length=11)
+
+    class Meta:
+        db_tables = 'user'
```

Comment:
class 명은 단수를 데이터베이스에 생성되는 테이블 명은 복수로 만드는 것이 컨벤션 입니다.

```suggestion
    class Meta:
        db_table = 'users'
```